### ISSUE TITLE
Tutorial: what is accessibility, and why is it important?

### ISSUE THREAD

**Issue by ironnysh** [WordPress/Learn#2370]
# Details
* Content type (Online Workshop, Lesson, Course, Tutorial, or Lesson Plan): Tutorial
* Content title: What is accessibility, and why is it important?
* Topic description: Introducing users to the topic of accessibility, explaining its importance, and helping them learn about best practices they can implement on their site.
* Audience (User, Developer, Designer, Contributor, etc.): User
* Experience Level (Beginner, Intermediate, Advanced, Any): Intermediate
 
## Learning Objectives
Gain a solid understanding of what is accessibility, and why is it essential.

# Related Resources and Other Notes

* Outline and initial discussion on #2188
* Part 2: #2201

### Automation Code
//tutorial

**Comment by ironnysh** [comment]
Hi @westnz, 
The draft of this script is also ready for review [in this doc](https://docs.google.com/document/d/1XpGTn5MviVYzND08eLpF__k0_nDJBeGzxuFIuR9F6fQ/edit?usp=sharing). 🥳

Looking forward to your feedback!

**Comment by westnz** [comment]
I have sent a request to access the doc 😉

**Comment by ironnysh** [comment]
Sorry, forgot to tick the box—done :-)

**Comment by westnz** [comment]
Reviewed 😃 
Great work @ironnysh

**Comment by ironnysh** [comment]
Hi @jonathanbossenger, now that the rewrite is done (thank you, @westnz! 🙌), I wanted to make sure that [this script](https://docs.google.com/document/d/1XpGTn5MviVYzND08eLpF__k0_nDJBeGzxuFIuR9F6fQ/edit) would work for the developer audience, too. Could you take a look?

**Comment by jonathanbossenger** [comment]
Hi @ironnysh just to make sure I understand, do you mean, "will this make sense to developers" or do you mean "should we use this in the developer learning pathway for #2188"

**Comment by ironnysh** [comment]
Hi @jonathanbossenger, option 2 😀 

As [discussed here](https://github.com/WordPress/Learn/issues/2188#issuecomment-1927032736):
> What if we stick to the "traditional" division of Developer vs. User used in the Learning Pathways? This way, both audiences **share the same intro**, but the practical part of the tutorial changes:

**User**
- [x] What is accessibility, and why is it important? (this issue)
- [x] Tools to test your content for accessibility. (#2201)

**Developer/designer**
- [x] What is accessibility, and why is it important? (this issue)
- [x] Best practices when developing/designing an accessible theme. (~To-Do~ #2188)
- [x] Tools to test your theme for accessibility. (#2201)

**Comment by jonathanbossenger** [comment]
@ironnysh Gotcha, thanks for clarifying (my memory fails me form time to time).

Yes, I think this will be perfect for the theme developer module on accessibility.

**Comment by ironnysh** [comment]
Coolio! :-)
I'll get to work on best practices for developers/designers, and incorporate the “tools to test” in there as well.

**Comment by jonathanbossenger** [comment]
Sounds great, I look forward to reading it.

**Comment by ervanyuff** [comment]
Hi team, I would like to be involved in recording this video

**Comment by westnz** [comment]
Thanks @ervanyuff 
I will be in contact once the script and voice recording is finished. 👍

**Comment by westnz** [comment]
The recording has been completed + shared.

**Comment by westnz** [comment]
![Text + Editor](https://github.com/WordPress/Learn/assets/89751452/8829306a-4216-4905-b2eb-334388cdaaeb)

**Comment by westnz** [comment]
Hey @ervanyuff 
When do you think the recording for this lesson will be done? 😀

**Comment by ervanyuff** [comment]
Hi @westnz This video might be finished earliest by the end of the next week. I'll update if I can finish sooner 🙏

**Comment by westnz** [comment]
Sounds good.  Thank you for the update. 👍

**Comment by ervanyuff** [comment]
Hi team, here is [what is accessibility, and why is it important](https://drive.google.com/file/d/1aZ3DXjgd0DZB9auTN77o4AbGUObpIMWA/view?usp=sharing) video. Please feel free to provide any feedback. Thank you 🙏

**Comment by westnz** [comment]
Fantastic video editing skills @ervanyuff 🌟 

Review:
1. I would add a question mark to the slide at 00:09?
2. Can the text start sooner to match the narration in three places: 1) 1:22, 2) 1:33, 3) 3:04. The narration begins slightly before the text appears.
3. The blurry example is not being displayed, 5:29 -5:35
4. I suggest using a red or yellow color for the annotation/rectangles to help guide the reader's eye, 6:00

**Comment by digitalchild** [comment]
Well done on the video edit @ervanyuff, agree with @westnz on the feedback.

**Comment by moltilearn** [comment]
hi all, currently I am at the contributor day at WCEU Torino and this is the first tutorial I have reviews, hope my feedback is useful.

## Tutorial/Lessons Review Checklist
Please tick all items you've confirmed:
- [x] Learning outcomes/objectives are clear.
- [x] Technical concepts introduced in the content are accurate.
- [x] The speed of demonstrations are easy to follow.
- [x] The narration audio matches what is shown visually.
- [ ] Spelling and grammar are correct.
- [x] Sound quality is consistent throughout the video.
- [ ] [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/) are being followed.
- [ ] Media assets are all in the public domain ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)).

Here are two comments: 
video time 2.26 talking about accessibility is hard
the image right is not clear to me, should this represent baking? I suggest finding a more distinct image.

Video time 4.11 table with accessibilities issues
maybe adding a title to the table such us issues would be helpful.

Cinzia

**Comment by hazeldrio** [comment]
## Tutorial/Lessons Review Checklist
Please tick all items you've confirmed:
- [x] Learning outcomes/objectives are clear.
- [x] Technical concepts introduced in the content are accurate.
- [x] The speed of demonstrations are easy to follow.
- [x] The narration audio matches what is shown visually.
- [x] Spelling and grammar are correct.
- [x] Sound quality is consistent throughout the video.
- [x] [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/) are being followed.
- [ ] Media assets are all in the public domain ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)).

Leave an additional comment below with feedback. 
1. Add Wordpress.org logo on title slide of video. 
2. Great video - Ervan!

**Comment by ervanyuff** [comment]
Hi team! Thank you for all the feedback @westnz @digitalchild @moltilearn @hazeldrio 

@moltilearn 2.26 is this icon is good enough?
![033-chocolate cake](https://github.com/WordPress/Learn/assets/164892023/41690bdf-b111-4c8b-85aa-6738761f524c)

@hazeldrio Did you mean by this part?
<img width="1611" alt="Screenshot 2024-06-14 at 10 08 26" src="https://github.com/WordPress/Learn/assets/164892023/7b22ac38-704e-4d59-8366-ccb7ffab5641">
I think the WordPress logo is already there. Do we need to add more? If so, maybe it can be put on the right side like this?
<img width="722" alt="Screenshot 2024-06-14 at 09 46 15" src="https://github.com/WordPress/Learn/assets/164892023/55fbfbbf-f27a-4d98-b645-785351c3054c">

**Comment by hazeldrio** [comment]
@ervanyuff Not a big deal but I think this is the new thumbnail graphic. So the call out was that the title slide was missing the full WP logo (see example below). 
![wp](https://github.com/WordPress/Learn/assets/172588005/31271922-e6cf-42af-9737-dc752cc50c7e)

**Comment by westnz** [comment]
Hey @hazeldrio and @ervanyuff 
There are a few thumbnail designs to choose from. Both of these are from the list of available ones: https://make.wordpress.org/training/2024/06/10/thumbnail-creation-project/ 😃

**Comment by ervanyuff** [comment]
Hi team! here is [the updated](https://drive.google.com/file/d/1Ver4_nh4in5GhwcWJKHzBm8a6wLdUh22/view?usp=sharing) video. Please let me know if there is anything missing :pray:

**Comment by westnz** [comment]
//published
